polus-csv-statistics-plugin/src/main.py

```diff
@@ -89,7 +89,25 @@ def kurtosis(data_list,data_dict):
     except ZeroDivisionError:
         data_dict['kurt'] = 'NaN'
     return data_dict
-    
+def iqr(data_list,data_dict):
+    if 'iqr  ' in data_dict.keys():
+        return
+    count(data_list, data_dict)
+    data_list.sort()
+    cnt=(int(data_dict['count']))//2
+    L_half = data_list[:cnt]
+    U_half = data_list[-cnt:]
+    data_dict['iqr'] = ((U_half[(len(U_half)-1)//2] + U_half[len(U_half)//2]) / 2)-((L_half[(len(L_half)-1)//2] + L_half[len(L_half)//2]) / 2)
```

Comment:
Separate this single line into three lines to make it easier to read and understand what's happening.
```python
q1 = ...
q3 = ...
data_dict['iqr'] = q3-q1
```